pkg/commands/game/lookForPlayers.go

```diff
@@ -0,0 +1,415 @@
+			Content:         fmt.Sprintf("<@&%s>",roleID),
+			Embed:           embed.MessageEmbed,
+			TTS:             false,
+			Files:           nil,
+			AllowedMentions: nil,
+			Reference:       nil,
+			File:            nil,
+		}
+		sentMessage, err = s.ChannelMessageSendComplex(inviteChannelID,message)
+	}else {
+		sentMessage, err = s.ChannelMessageSendEmbed(inviteChannelID,embed.MessageEmbed)
+	}
+	if err != nil {
+		return errors.New(fmt.Sprintf("Error sending embed message: %v", err))
+	}
+	s.MessageReactionAdd(sentMessage.ChannelID, sentMessage.ID, "👋")
+	s.MessageReactionAdd(sentMessage.ChannelID, sentMessage.ID, "🗑️")
+	s.MessageReactionAdd(sentMessage.ChannelID, sentMessage.ID, "🎮")
+	return nil
+}
+
+func (l *LookCommand) sendInteractionResponse(s *discordgo.Session, i *discordgo.InteractionCreate, content string){
+	err := s.InteractionRespond(i.Interaction, &discordgo.InteractionResponse{
+		Type: discordgo.InteractionResponseChannelMessageWithSource,
+		Data: &discordgo.InteractionApplicationCommandResponseData{
+			Content: content,
+		},
+	})
+	if err != nil {
+		s.ChannelMessageSend(i.ChannelID, fmt.Sprintf("Error sending interaction response: %v", err))
+		log.Println(err)
+		return
+	}
+}
+
+func (l *LookCommand) handleReactionAdd(s *discordgo.Session, r *discordgo.MessageReactionAdd) {
+	message, err := s.ChannelMessage(r.ChannelID, r.MessageID)
+	if err != nil {
+		return
+	}
+
+	if !l.checkEmbed(s,message){
+		return
+	}
+
+	hostID,currentPlayers,backupPlayers,neededPlayers:=l.getPlayers(s,message)
+
+	if r.Emoji.MessageFormat() == "👋"{
+		l.handleJoinReaction(currentPlayers,backupPlayers,message,s)
+		if message.Embeds[0].Fields[2].Value=="Now!" && len(currentPlayers) >= neededPlayers{
+			l.startGame(s, r, currentPlayers, message, hostID, err)
+		}
+	}
+
+	if r.Emoji.MessageFormat() == "🗑️"{
+		if r.UserID == hostID{
+			//Notify players
+			l.messagePlayers(s, r, currentPlayers, fmt.Sprintf("The invite for %s has been cancelled.", message.Embeds[0].Title))
+		}
+	}
+
+	if r.Emoji.MessageFormat() == "🎮" {
+		if r.UserID == hostID {
+			l.startGame(s, r, currentPlayers, message, hostID, err)
+		}
+	}
+}
+
+func (l *LookCommand) handleReactionRemove(s *discordgo.Session, r *discordgo.MessageReactionRemove) {
+	message, err := s.ChannelMessage(r.ChannelID, r.MessageID)
+	if err != nil {
+		return
+	}
+	if !l.checkEmbed(s,message){
+		return
+	}
+
+	_,currentPlayers,backupPlayers, _:=l.getPlayers(s,message)
+
+	if r.Emoji.MessageFormat() ==  "👋"{
+		l.handleJoinReaction(currentPlayers,backupPlayers,message,s)
+	}
+}
+
+func (l *LookCommand) checkEmbed(s *discordgo.Session, message *discordgo.Message)bool{
+	if message.Author.ID != s.State.User.ID {
+		return false // not the bot user
+	}
+	if len(message.Embeds) < 1 {
+		return false // not an embed
+	}
+	if !(len(message.Embeds[0].Fields) >= 5) {
+		return false // not the correct embed
+	}
+
+	if !(message.Embeds[0].Fields[0].Name == "Host") {
```

Comment:
```suggestion
	if message.Embeds[0].Fields[0].Name != "Host"){
```

you okay?

Reply:
I have a habit of making things harder than they should be appearantly 😄